cirq/experiments/t2_decay_experiment.py

```diff
@@ -70,8 +71,20 @@ def t2_decay(
     The same method of measuring the final state as Ramsey experiment is applied
     after the second half of the delay period.
 
-    CPMG, or the Carr-Purcell-Meiboom-Gill sequence, is currently not
-    implemented.
+    CPMG, or the Carr-Purcell-Meiboom-Gill sequence, involves sending a sequence
+    of pi pulses (X gates) in a specific timing pattern:
+    π/2, t, π, 2t, π, ... 2t, π, t
```

Comment:
I just included the final reference, since that is what we are actually doing.  I also expanded the explanation.

I have not found a good paper that summarizes the procedure from a quantum computing perspective, which is sad for poor computer science people like me :(